### PR TITLE
[DONE] Madeleine rescale styling checkbox

### PR DESCRIPTION
**Pull request by GeOdin**

### CODE REVIEW

**Comment by GeOdin** [comment]
Review and merge
https://github.com/nens/lizard-management-client/pull/132
before this one.

File: src/data_management/rasters/RasterForm.js [hunk 1]
```diff
@@ -554,7 +573,8 @@ class RasterFormModel extends Component {
       supplierCode: currentRaster.supplier_code,
       observationType: currentRaster.observation_type,
       sharedWith: currentRaster.shared_with,
-      accesModifier: currentRaster.access_modifier
+      accesModifier: currentRaster.access_modifier,
```
Comment (remcogerlich) [inline src/data_management/rasters/RasterForm.js]: Typo? acces -> access

File: src/data_management/rasters/RasterForm.js [hunk 2]
```diff
@@ -492,15 +506,17 @@ class RasterFormModel extends Component {
...
+        rescalable: false
       },
       options: {},
       aggregationType: "", // choice: none | counts | curve | histogram | sum | average
       supplierId: supplierId,
       supplierCode: "",
       observationType: null,
       sharedWith: [],
-      accesModifier: "Private"
+      accesModifier: "Private",
```
Comment (remcogerlich) [inline src/data_management/rasters/RasterForm.js]: idem

File: src/data_management/rasters/RasterForm.js (hunk 1 above)
Comment (GeOdin) [reply src/data_management/rasters/RasterForm.js]: Goed gezien, ik had zelf alleen de komma aangepast, vandaar dat ik het niet had gezien.

File: src/utils/rasterOptionFunctions.js [hunk 3]
```diff
@@ -171,6 +171,10 @@ export function validateStyleObj(style) {
   } else {
     return false;
   }
+
```
Comment (Derryrover) [inline src/utils/rasterOptionFunctions.js]: This code will never be reached.
Is also not needed.
Can be removed

**Pull request merged**